.github/workflows/publish-example-images.yaml

```
@@ -33,12 +32,25 @@ jobs:
           - component-name: tf-smoke
             dockerfile: examples/tensorflow/tf_sample/Dockerfile
           - component-name: pytorch-dist-sendrecv-test
+            platforms: linux/amd64
```

```suggestion
            platforms: linux/amd64,linux/arm64
```

We could specify the multiple platforms like this.
Could you apply the same approach in other places?

---

Yeah of course!